example/vm/config.yaml

```diff
@@ -0,0 +1,201 @@
+# Last Collector-Contrib Validation: v0.96.0
+receivers:
+    prometheus/self:
```

Comment:
Will everyone be using Prom to monitor the Collector? Maybe we need some explanation here about what this does?

Reply:
yeah would be good to explain why we do this for now. In the future users won't need to do this, but for now they do